src/modules/home/components/LoggedOutHome/LoggedOutHome.module.scss

```diff
@@ -51,6 +51,30 @@ $c-hero-access-card-min-width: 57.2rem;
 		}
 	}
 
+	@keyframes pulse {
+		0% {
+			transform: translateY(0);
+			opacity: 0.3;
+		}
+
+		50% {
+			transform: translateY(5px);
+			opacity: 1;
+		}
+
+		100% {
+			transform: translateY(0);
+			opacity: 0.3;
+		}
+	}
+
+	&__arrow-down {
+		display: block;
+		font-size: 4rem;
+		line-height: 2;
+		animation: pulse 3s infinite ease-in-out;
+	}
+
```

Comment:
Just for fun:
Je kan dit ook als volgt schrijven
```
@keyframes pulse {
      from {
	      transform: translateY(0);
	      opacity: 0.3;
      }
      
      to {
	      transform: translateY(5px);
	      opacity: 1;
      }
}

animation: pulse 1.5s infinite alternate ease-in
```

Ik zou hier voor een `ease-in` gaan omdat dit een bouncier (imo meer natuurlijk) effect heeft.
Een font-size van 3rem lijkt me hier ook the way to go omdat het pijltje dan minder zwaar is tegenover de tekst.

☝️ Uiteindelijk allemaal personal preference dus doe ermee wat je wilt!

https://user-images.githubusercontent.com/85545258/172624720-6fe66fe8-b73d-40bb-98a5-1d5ae4bb0ef5.mp4

Reply:
nice, en het is minder bytes :p